discord_api.py

```diff
@@ -124,11 +126,13 @@ def help(message):
 	if author_restrict == True:
 		msg += '!type <mensagem> - Envia uma mensagem para rodapé (só os users autorizados podem adicionar mensagens)\n'
 		msg += '!type rm <id> - Remove uma mensagem de rodapé (só os users autorizados podem eliminar mensagens)\n'
+		msg += '!type id - Mostra os ids das mensagens\n'
+		msg += '!type stop - Desliga o bot (só os users autorizados podem desligar o bot)'
 	else:
 		msg += '!type <mensagem> - Envia uma mensagem para rodapé (todos os users podem adicionar mensagens)\n'
 		msg += '!type rm <id> - Remove uma mensagem de rodapé (todos os users podem eliminar mensagens)\n'
-	msg += '!type id - Mostra os ids das mensagens\n'
-	msg += '!type stop - Desliga o bot'
+		msg += '!type id - Mostra os ids das mensagens\n'
+		msg += '!type stop - Desliga o bot'
 	await channel_obs.send(msg)
 
 client.run(os.getenv('DISCORD_TOKEN'))
```

Comment:
You're missing a new line at the end of the file. Probably best to add an `.editorconfig` file with some sane defaults to prevent this.

Reply:
You haven't sorted the missing new line, yet.

Reply:
The missing new line is already corrected, as you can see in line 123 of [discord_api.py](https://github.com/vostpt/DISOBS-BOT/blob/fix/DISOBS-BOT-6/discord_api.py)

Reply:
![image](https://user-images.githubusercontent.com/1413966/57840358-609f7580-77c0-11e9-975e-d77cde9fb93f.png)
Seems to be missing, still.

Reply:
Sorry, I thought that you were talking about msg strings. I'll fix that and commit again